docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -50,15 +50,19 @@ export default class ComponentExample extends Component {
       marginLeft: '0.25em',
     }
 
+    const codeIconStyle = {
+      fontSize: '1.5em',
+      fontWeight: 'bold',
+    }
```

Comment:
Cleaner code icon more inline with SUI docs.  Also removed the button.

![image](https://cloud.githubusercontent.com/assets/5067638/16722505/eab70ec0-46fb-11e6-8afd-a11992df1c01.png)